opentok-layout.js.map

```diff
@@ -1 +0,0 @@
.call(\n        container.querySelectorAll('#' + id + '>*:not(.' + opts.bigClass + ')'),\n        filterDisplayNone);\n\n    if (bigOnes.length > 0 && smallOnes.length > 0) {\n      var bigWidth, bigHeight;\n\n      if (availableRatio > getVideoRatio(bigOnes[0])) {\n        // We are tall, going to take up the whole width and arrange small\n        // guys at the bottom\n        bigWidth = Width;\n        bigHeight = Math.floor(Height * opts.bigPercentage);\n        offsetTop = bigHeight;\n        bigOffsetTop = Height - offsetTop;\n      } else {\n        // We are wide, going to take up the whole height and arrange the small\n        // guys on the right\n        bigHeight = Height;\n        bigWidth = Math.floor(Width * opts.bigPercentage);\n        offsetLeft = bigWidth;\n        bigOffsetLeft = Width - offsetLeft;\n      }\n      if (opts.bigFirst) {\n        arrange(bigOnes, bigWidth, bigHeight, 0, 0, opts.bigFixedRatio, opts.bigMinRatio,\n          opts.bigMaxRatio, opts.animate);\n        arrange(smallOnes, Width - offsetLeft, Height - offsetTop, offsetLeft, offsetTop,\n          opts.fixedRatio, opts.minRatio, opts.maxRatio, opts.animate);\n      } else {\n        arrange(smallOnes, Width - offsetLeft, Height - offsetTop, 0, 0, opts.fixedRatio,\n          opts.minRatio, opts.maxRatio, opts.animate);\n        arrange(bigOnes, bigWidth, bigHeight, bigOffsetLeft, bigOffsetTop,\n          opts.bigFixedRatio, opts.bigMinRatio, opts.bigMaxRatio, opts.animate);\n      }\n    } else if (bigOnes.length > 0 && smallOnes.length === 0) {\n      // We only have one bigOne just center it\n      arrange(bigOnes, Width, Height, 0, 0, opts.bigFixedRatio, opts.bigMinRatio,\n        opts.bigMaxRatio, opts.animate);\n    } else {\n      arrange(smallOnes, Width - offsetLeft, Height - offsetTop, offsetLeft, offsetTop,\n        opts.fixedRatio, opts.minRatio, opts.maxRatio, opts.animate);\n    }\n  };\n\n  var initLayoutContainer = function(container, opts) {\n    opts = defaults(opts || {}, {\n      maxRatio: 3 / 2,\n      minRatio: 9 / 16,\n      fixedRatio: false,\n      animate: false,\n      bigClass: 'OT_big',\n      bigPercentage: 0.8,\n      bigFixedRatio: false,\n      bigMaxRatio: 3 / 2,\n      bigMinRatio: 9 / 16,\n      bigFirst: true\n    });\n    container = typeof(container) === 'string' ? document.querySelector(container) : container;\n\n    if (document.readyState === 'complete') {\n      layout(container, opts);\n    } else {\n      window.addEventListener('load', function() {\n        layout(container, opts);\n      });\n    }\n\n    return {\n      layout: layout.bind(null, container, opts)\n    };\n  };\n\n  if (typeof module === 'undefined' || typeof module.exports === 'undefined') {\n    window.initLayoutContainer = initLayoutContainer;\n  } else {\n    module.exports.initLayoutContainer = initLayoutContainer;\n  }\n\n})((typeof window !== 'undefined' && window.hasOwnProperty('jQuery')) ? window.jQuery : undefined);\n"],"sourceRoot":""}
```

Comment:
I deleted this file and it was not regenerated in the build... where did it come from originally? 
Is a map needed for the non-minimized file? 🤔

Reply:
Ha, no good point I don't think it's needed and you're right webpack doesn't generate one. My only guess is that I messed up and accidentally generated a map with the wrong file name and committed that file.